src/public/modules/forms/admin/components/form-emails-input.client.component.js

```diff
@@ -9,29 +9,32 @@ angular.module('forms').component('formEmailsInputComponent', {
     saveForm: '&',
   },
   controllerAs: 'vm',
-  controller: formEmailsInputController,
+  controller: ['$scope', formEmailsInputController],
 })
 
-function formEmailsInputController() {
+function formEmailsInputController($scope) {
   const vm = this
-  vm.validateEmails = (emails) => {
-    if (!emails) {
-      vm.emailErrorMsg =
-        'You must at least enter one email to receive responses'
-      return
-    }
 
-    const err = checkForErrors(String(emails).split(','))
+  $scope.$watch('vm.formData.emails', (newEmails) => {
+    vm.emailInfoMsg =
```

Comment:
instead of adding a custom `$watch`, might it be better to add this logic in `checkForErrors` or `vm.validateEmails`? Then at least all the code which runs on changes to the emails input is all in one place

Reply:
i want the initial load to also show the info message. Since `vm.validateEmails` only run on change, it is not suitable. I also tried just assigning to vm.emailInfoMsg directly, but vm.formData.emails may be undefined at the start and won't get updated when formData.email finally gets populated.